generators/app/index.js

```diff
@@ -27,7 +27,7 @@ module.exports = yeoman.generators.Base.extend({
   },
 
   install: function () {
-    if (this.options['skip-install']) {
+    if (this.options['skip-install'] || this.options.skipInstall) {
       return;
     }
 
```

Comment:
there isn't a `bower.json` file created here so expecting one to exist may cause problems